requirements/dev.txt

```diff
@@ -1,10 +1,11 @@
 #
-# This file is autogenerated by pip-compile with python 3.6
+# This file is autogenerated by pip-compile with python 3.9
```

Comment:
[on tox.ini file](https://github.com/kytos-ng/coloring/blob/master/tox.ini#L17) it's still using `py36` there could you update accordinly to `py39`? (three lines to be updated there)